compiler/luci-interpreter/src/kernels/UnidirectionalSequenceLSTM.test.cpp

```diff
@@ -39,26 +39,333 @@ class UnidirectionalSequenceLSTMTest : public ::testing::Test
+  Shape input_to_input_weights_shape{n_cell, n_input};
+  Tensor input_to_input_weights_tensor = makeInputTensor<DataType::FLOAT32>(
+    input_to_input_weights_shape, input_to_input_weights, _memory_manager.get());
+
+  Tensor output_tensor = makeOutputTensor(DataType::FLOAT32);
+  Tensor scratchpad_1(DataType::FLOAT32, Shape({}), {}, "");
+  Tensor scratchpad_2(DataType::FLOAT32, Shape({}), {}, "");
+  Tensor scratchpad_3(DataType::FLOAT32, Shape({}), {}, "");
+
+  UnidirectionalSequenceLSTMParams params{};
+  params.activation = Activation::TANH;
+  params.cell_clip = 0.0;
+  params.proj_clip = 0.0;
+  params.time_major = true;
+  params.asymmetric_quantize_inputs = false;
+
+  // clang-format off
+  UnidirectionalSequenceLSTM kernel(
+    &input_tensor,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    nullptr, nullptr, nullptr,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    nullptr, nullptr,
+    &input_to_input_weights_tensor,
+    &input_to_input_weights_tensor,
+    nullptr, nullptr, nullptr, nullptr,
+    &output_tensor, &scratchpad_1, &scratchpad_2, &scratchpad_3,
+    params);
+  // clang-format on
+
+  EXPECT_ANY_THROW(kernel.configure());
 }
 
 TEST_F(UnidirectionalSequenceLSTMTest, Invalid_Input_Shape_2_NEG)
 {
-  // TODO implement
-  SUCCEED();
+  const int32_t n_batch = 1;
+  const int32_t n_input = 2;
+  const int32_t n_cell = 4;
+  const int32_t n_output = 4;
+  const int32_t sequence_length = 3;
+
+  std::vector<float> input_data{2., 3., 3., 4., 1., 1.};
+  Shape input_shape{sequence_length, n_batch, n_input};
+  Tensor input_tensor =
+    makeInputTensor<DataType::FLOAT32>(input_shape, input_data, _memory_manager.get());
+
+  std::vector<float> input_to_input_weights = {-0.45018822, -0.02338299, -0.0870589,  -0.34550029,
+                                               0.04266912,  -0.15680569, -0.34856534, 0.43890524};
+  Shape input_to_input_weights_shape{n_cell, n_input};
+  Tensor input_to_input_weights_tensor = makeInputTensor<DataType::FLOAT32>(
+    input_to_input_weights_shape, input_to_input_weights, _memory_manager.get());
+
+  Tensor output_tensor = makeOutputTensor(DataType::FLOAT32);
+  Tensor scratchpad_1(DataType::FLOAT32, Shape({}), {}, "");
+  Tensor scratchpad_2(DataType::FLOAT32, Shape({}), {}, "");
+  Tensor scratchpad_3(DataType::FLOAT32, Shape({}), {}, "");
+
+  UnidirectionalSequenceLSTMParams params{};
+  params.activation = Activation::TANH;
+  params.cell_clip = 0.0;
+  params.proj_clip = 0.0;
+  params.time_major = true;
+  params.asymmetric_quantize_inputs = false;
+
+  // clang-format off
```

Comment:
May I ask why clang-format is turned off?

Reply:
Ah... with format,
```cxx
  UnidirectionalSequenceLSTM kernel(
    &input_tensor, &input_to_input_weights_tensor, &input_to_forget_weights_tensor,
    &input_to_cell_weights_tensor, &input_to_output_weights_tensor,
    &recurrent_to_input_weights_tensor, &recurrent_to_forget_weights_tensor,
    &recurrent_to_cell_weights_tensor, &recurrent_to_output_weights_tensor, nullptr, nullptr,
    nullptr, &input_gate_bias_tensor, &forget_gate_bias_tensor, &cell_gate_bias_tensor,
    &output_gate_bias_tensor, nullptr, nullptr, &output_state_tensor, &cell_state_tensor, nullptr,
    nullptr, nullptr, nullptr, &output_tensor, &scratchpad_1, &scratchpad_2, &scratchpad_3, params);
```
which I had difficulties reading the code while debugging :)
I think it's OK to remove this.